internal/cli/atlas/livemigrations/cutover.go

```diff
@@ -43,12 +43,12 @@ func (opts *CutoverOpts) initStore(ctx context.Context) func() error {
 }
 
 func (opts *CutoverOpts) Run() error {
-	r, err := opts.store.CreateLiveMigrationCutover(opts.ConfigProjectID(), opts.liveMigrationID)
+	err := opts.store.CreateLiveMigrationCutover(opts.ConfigProjectID(), opts.liveMigrationID)
 	if err != nil {
 		return err
 	}
 
-	return opts.Print(r)
+	return opts.Print(nil)
```

Comment:
nit the same result as not printing/no statement

Reply:
It's consistent with the pattern of other commands with no-response so I'm ok with leaving it as is 🤷

Reply:
No worries.